README.md

```diff
@@ -148,6 +148,8 @@ for more details on the structure of the metadata file.
   file names will be the same as they are on Pivotal Network - e.g. a file with
   name `some-file.txt` will be downloaded to `/tmp/build/get/some-file.txt`.
 
+* `unpack`: *Optional.* Whether to unpack the downloaded file
```

Comment:
Could we see a little more clarity on what "unpack" means here? Specifically, some example use-cases, supported file-types, etc, would be really useful.

Reply:
There is an effort to distribute both concourse tasks (delivered as a .zip) and docker rootfs (image delivered as .tgz).  Without unpack feature I would need to have a custom task that takes in files from pivnet resource as input and then just unzip/untar these files to pass to something else.  Technically this would work fine for the tasks but the image has to be passed as output of `get` of the resource to be able to replace which image tasks of a given job run with.